docs/mdbook/SUMMARY.md

```diff
@@ -5,11 +5,12 @@
 # User guide
 
 - [Installation](./user-guide/install.md)
-- [Helpers](./user-guide/helpers.md)
 - [FAQ](./user-guide/faq.md)
 - [Configuration examples](./user-guide/config-examples.md)
 - [Lazy Loading](./user-guide/lazy-loading.md)
 
+@FUNCTIONS_MENU@
+
```

Comment:
I originally put this higher up because the old `helpers.md` guide was near the top. However, it'd also be valid to have "Functions Reference" section _after_ the "Options" reference section, if preferred.

Thoughts?